packages/clay-css/src/scss/variables/_management-bar.scss

```diff
@@ -76,8 +76,8 @@ $management-bar-primary: map-deep-merge(
 					color: $navbar-light-active-color,
 				),
 				disabled: (
-					disabled-color: $navbar-light-disabled-color,
-					disabled-opacity: 1,
```

Comment:
@matuzalemsteles I don't think I understand the question.
1) The button should look disabled and can be clicked?
2) The button should look disabled and cannot be clicked?

Reply:
Oh sorry for not being clear, well what I'm saying is, the quick action will have the property of `disabled`, that is, we don't want to have an action for the button but for me, leave the opacity different from the disabled state of a button sounds confusing. In the end, it's like the button is not disabled but it is, this is confusing.

Reply:
Ah I see, the disabled state is just a change in color. We can use opacity or style it directly. We are styling it directly here because I wanted to match Lexicon's color exactly. If we want to use opacity we can.

Reply:
I see, I'm more wondering why this behavior is wanted here and specifically for the quick action which appears to be true coming from the lexicon, maybe @victorg1991 can give us a little more context here about the problem, I'm still a little confused about this the problem.

Reply:
Clay wasn't providing these styles due to an error I made converting to the new variable pattern. I think this is the main problem, I just didn't understand the info previously. The opacity is just blocking the possibility of inheriting a disabled opacity value from the base class `nav-link`, since the disabled color is correct.

Reply:
Yes, CSS makes sense, I'm just confused by the visuals because the state of `disabled` is going to be 1 instead of 0.4.

Reply:
Ok, thanks @victorg1991!!